src/sqlfluff/dialects/dialect_databricks.py

```diff
@@ -319,6 +319,125 @@ class AlterDatabaseStatementSegment(sparksql.AlterDatabaseStatementSegment):
     )
 
 
+class VolumeReferenceSegment(ansi.ObjectReferenceSegment):
+    """Volume reference."""
+
+    type = "volume_reference"
+
+
+class AlterVolumeStatementSegment(BaseSegment):
+    """Alter Volume Statement.
+
+    https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-alter-volume.html
+    """
+
+    type = "alter_volume_statement"
+
+    match_grammar = Sequence(
+        "ALTER",
+        "VOLUME",
+        Ref("VolumeReferenceSegment"),
+        OneOf(
+            Sequence(
+                "RENAME",
+                "TO",
+                Ref("VolumeReferenceSegment"),
+            ),
+            Ref("SetOwnerGrammar"),
+            Ref("SetTagsGrammar"),
+            Ref("UnsetTagsGrammar"),
+        ),
+    )
+
+
+class CreateVolumeStatementSegment(BaseSegment):
+    """Create Volume Statement.
+
+    https://docs.databricks.com/en/sql/language-manual/sql-ref-syntax-ddl-create-volume.html
+    """
+
+    type = "create_volume_statement"
+
+    match_grammar = OneOf(
+        # You can create a non-external volume without a location
+        Sequence(
+            "CREATE",
+            "VOLUME",
+            Ref("IfNotExistsGrammar", optional=True),
+            Ref("VolumeReferenceSegment"),
+            Ref("CommentGrammar", optional=True),
+        ),
+        # Or you can create an external volume that must have a location
+        Sequence(
+            "CREATE",
+            "EXTERNAL",
+            "VOLUME",
+            Ref("IfNotExistsGrammar", optional=True),
+            Ref("VolumeReferenceSegment"),
+            Ref("LocationGrammar"),
+            Ref("CommentGrammar", optional=True),
+        ),
+    )
+
+
+class DescribeVolumeStatementSegment(BaseSegment):
```

Comment:
We generally don't go this granular on statement separation. We have a `DescribeStatementSegment` and `ShowStatementSegment`, which covers all of the describes and shows. You'll want to add to this, there is one of each defined in Spark

Reply:
Should I copy each of those into the Databricks dialect and add these?

Only reason I didn't was so I wasn't duplicating the existing sparksql show/describe statements, but rather do a pseudo-extension of them. But if we'd prefer to copy over + modify I can do that as well.

Reply:
Also this may be an item for future cleanup, but looking at the dialect_sparksql.py files, it looks like the describe behavior is broken out a few different ways (e.g., `DESCRIBE HISTORY` and `DESCRIBE DETAIL`).

This isn't directly relevant to how we want to handle this case, but may want to be standardized in the future to avoid confusion.

Reply:
There's a syntax for "copy and modify" which should be what you're looking for. It can't do everything, but it can do a certain amount of insertions and removals. Do a `crtl+f` for `.copy(` in a few of the dialect files and you'll see how it's used in various ways.

Reply:
Okay I think I've done it in a more standardized way. I had to convert a few things to grammars as you can copy and insert new options into those, whereas segment's copy does not support inserts. So hopefully I've done it correctly.

Reply:
I also found some of the show commands in Databricks are SLIGHTLY different from their Spark lineage, so I've updated those while there and documented as well as I can.